docs/userguide.rst

```diff
@@ -264,9 +286,9 @@ the ``reana-client start`` or ``reana-client run`` commands.
 
 .. code-block:: console
 
-   $ reana-client start -w workflow.1 -o target='gendata'
+   $ reana-client start -w myanalysis.42 -o --target=gendata
```

Comment:
Ah, indeed, we are using engine options as such, so dashes included.

(This raises a question about possibly harmonising these across engines, e.g. -o TARGET meaning --target in CWL world, and something else in Serial world, and something else in Snakemake world.... This would allow us to use nicer word pairs for Serial maybe? (`from ... until ...` would be a very natural English option combination. We can muse on this later...